### PR TITLE
fix a comment

### PR DESCRIPTION
**Pull request by cocoa-maemae**

### CODE REVIEW

File: azure/tfc-workspace.tf [hunk 1]
```diff
@@ -6,7 +6,7 @@ provider "tfe" {
 }
 
 # Runs in this workspace will be automatically authenticated
-# to Azure with the permissions set in the Azure policy. TODO: che k if wording right
+# to Azure with the permissions set in the Azure policy. TODO: check if wording right
```
Comment (mpminardi) [inline azure/tfc-workspace.tf:9]: Whoops, good catch here! I think we can remove this TODO entirely though:

```suggestion
# to Azure with the permissions set in the Azure policy.
```

**Comment by mpminardi** [comment]
Thanks for the contributions @cocoa-maemae! It looks like your first commit was not associated with a GitHub account that has signed the CLA. Are you able to rebase and have both commits associated to the email you use for `cocoa-maemae` here?

**Comment by cocoa-maemae** [comment]
Hi @mpminardi,
Now I rebased the commits and fixed as recommended.
Would you check them again?

**Review by mpminardi — APPROVED** [review]
Looks good, thanks again @cocoa-maemae!

**Pull request merged**